sub.js

```diff
@@ -0,0 +1,5 @@
+const sub = (num1, num2) => {
+    return num1 - num2
+};
+
+module.exports = sub;
```

Comment:
I am writing this here because I am writing this to fit in the comment character count.